tools-v2/pkg/cli/command/curvebs/list/scanstatus/scanstatus.go

```diff
@@ -0,0 +1,123 @@
+/*
+*  Copyright (c) 2023 NetEase Inc.
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+ */
+/*
+* Project: curve
+* Created Date: 2023-05-18
+* Author: lianzhanbiao
+ */
+
+package scanstatus
+
+import (
+	"fmt"
+
+	cmderror "github.com/opencurve/curve/tools-v2/internal/error"
+	cobrautil "github.com/opencurve/curve/tools-v2/internal/utils"
+	basecmd "github.com/opencurve/curve/tools-v2/pkg/cli/command"
+	"github.com/opencurve/curve/tools-v2/pkg/cli/command/curvebs/list/copyset"
+	"github.com/opencurve/curve/tools-v2/pkg/config"
+	"github.com/opencurve/curve/tools-v2/pkg/output"
+	"github.com/opencurve/curve/tools-v2/proto/proto/common"
+	"github.com/spf13/cobra"
+)
+
+const (
+	copysetExample = `$ curve bs list scan-status`
+)
+
+type ScanStatusCommand struct {
+	CopysetInfoList []*common.CopysetInfo
+	basecmd.FinalCurveCmd
+}
+
+var _ basecmd.FinalCurveCmdFunc = (*ScanStatusCommand)(nil) // check interface
+
+func NewListScanStatusCommand() *ScanStatusCommand {
+	scanStatusCommand := &ScanStatusCommand{
+		FinalCurveCmd: basecmd.FinalCurveCmd{
+			Use:     "scan-status",
+			Short:   "list curvebs all copyset that scanning is false",
+			Example: copysetExample,
+		},
+	}
+
+	basecmd.NewFinalCurveCli(&scanStatusCommand.FinalCurveCmd, scanStatusCommand)
+	return scanStatusCommand
+}
+
+func NewScanStatusCommand() *cobra.Command {
+	return NewListScanStatusCommand().Cmd
+}
+
+func (cCmd *ScanStatusCommand) AddFlags() {
+	config.AddRpcTimeoutFlag(cCmd.Cmd)
+	config.AddRpcRetryTimesFlag(cCmd.Cmd)
+	config.AddBsMdsFlagOption(cCmd.Cmd)
+}
+
+func (cCmd *ScanStatusCommand) Init(cmd *cobra.Command, args []string) error {
+	mdsAddrs := config.GetBsFlagString(cCmd.Cmd, config.CURVEBS_MDSADDR)
+	timeout := config.GetFlagDuration(cCmd.Cmd, config.RPCTIMEOUT)
+	retrytimes := config.GetFlagInt32(cCmd.Cmd, config.RPCRETRYTIMES)
+
+	cCmd.Cmd.ParseFlags([]string{
```

Comment:
sub-command can reuse flag without it? Because we have added flags in `AddFlags`.

Reply:
> sub-command can reuse flag without it? Because we have added flags in `AddFlags`.

I just delete these lines ant it seems ok.